brownfield_django/main/views.py

```diff
@@ -310,6 +310,18 @@ def get(self, request):
         return HttpResponseRedirect(url)
 
 
+class ArchiveCourseView(CSRFExemptMixin, JSONResponseMixin, View):
+
+    def get(self, request, pk):
+        crs = Course.objects.get(pk=pk)
+        crs.archive = True
+        crs.save()
+        if crs.archive is True:
```

Comment:
This test is redundant. crs.archive has just been set to True a few lines previously.